flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1254,6 +1254,7 @@ under the License.
 									<include>org.apache.parquet:parquet-encoding</include>
 									<include>org.apache.parquet:parquet-format-structures</include>
 									<include>org.apache.parquet:parquet-jackson</include>
+									<include>it.unimi.dsi:fastutil</include>
```

Comment:
See comments above, we don't need to introduce this dependency.

Reply:
done